common/src/main/scala/io/phdata/services/EmailService.scala

```diff
@@ -31,12 +31,15 @@ class EmailServiceImpl[F[_]: Effect](context: AppContext[F], workspaceService: W
       fromAddress = context.appConfig.smtp.fromEmail
       to <- context.lookupLDAPClient.findUserByDN(memberRoleRequest.distinguishedName)
       toAddress <- OptionT(Effect[F].pure(to.email))
+      owner <- context.lookupLDAPClient.findUserByDN(workspace.requestedBy)
       values = Map(
         "roleName" -> memberRoleRequest.role.get.show,
         "resourceType" -> memberRoleRequest.resource,
         "workspaceName" -> workspace.name,
         "uiUrl" -> resolveUiUrl,
-        "workspaceId" -> workspaceId
+        "workspaceId" -> workspaceId,
+        "ownerName" -> owner.name,
+        "ownerEmail" -> owner.email.getOrElse("Email not provided")
```

Comment:
Print out error message if there was one, or print out error saying the owner email was not found